task3.2.py

```diff
@@ -0,0 +1,14 @@
+'''Напишите программу, которая найдёт произведение пар чисел списка.
+Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+Пример:
+- [2, 3, 4, 5, 6] => [12, 15, 16];
+- [2, 3, 5, 6] => [12, 15]'''
+
+lst1 = [2, 3, 4, 5, 6]
+lst2 = [2, 3, 5, 6]
+new_lst = []
+product =0
+for i in range(0, (len(lst2) + 1) // 2):
+    product = lst2[i] * (lst2[len(lst2) - 1 - i])
```

Comment:
```suggestion
    product = lst2[i] * (lst2[-1 - i])
```
Можно и так. От этого ничего не изменится.